sys/net/ble/skald/Kconfig

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_SKALD
+    bool "Configure SKALD"
+    depends on MODULE_SKALD
+    help
+        Configure Skald, BLE advertising stack, using Kconfig.
+
+if KCONFIG_MODULE_SKALD
+
+config SKALD_INTERVAL
+    int "Advertising interval in microseconds"
+    default 1000000
+    help
+        Configure advertising interval in microseconds. Default value is 1
+        second which is 1000000 microseconds.
+
+config SKALD_ADV_CHAN_EN
+    bool "Configure advertising channels"
+    help
+        Configure advertising channels namely 37, 38 and 39.
```

Comment:
Why is this switch needed?

Reply:
That was to hide the choice and to show it only if the user intends to configure the advertising channels.

Reply:
But in the case the user does not want to configure them, they could be left untouched right? I don't think it adds value to have an extra step to see what can be configured.

Reply:
If you say so. Yes, I shall skip the same.